frameworks/Rust/tokio-minihttp/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "tokio-minihttp"
+version = "0.1.0"
+authors = ["Gökberk Yaltıraklı <webdosusb@gmail.com>"]
+
+[dependencies]
+futures = "0.1"
+httparse = "1.1"
+serde_json = "0.8"
+num_cpus = "1.0"
+tokio-proto = { git = "https://github.com/tokio-rs/tokio-proto" }
+tokio-service = { git = "https://github.com/tokio-rs/tokio-service" }
```

Comment:
hi @gkbrk thanks for the submission!
If you're using git repos as a source they need to be tied down to specific commits or releases